packages/util/test/errors.test.ts

```diff
@@ -88,9 +88,9 @@ describe('FirebaseError', () => {
     try {
```

Comment:
Is this the only file in util that had errors? There are a few try/catch blocks in util/src so I'm kind of surprised.

Reply:
There are more files to change. This is just an initial attempt